### DISCUSSION TITLE
How to make a build with EraserBrush

### DISCUSSION THREAD

**Discussion by dilankamr** [fabricjs/fabric.js#8262] [General]
I have customized this repo by forking. So, I am making a custom build for my application. I need to make a build with 'EraserBrush' support. How can I build a new min.js file with '**EraserBrush**'? What is the command for that build?  

Thanks

**Comment by ShaMan123** [comment]
master is WIP
Eraser isn't part of the build yet.
You can pull it in by importing the file in `index.js`

I don't know what your purposes are but the changes you applied to the source code can be made at runtime, eliminating the need for maintaining a fork.

**Reply by dilankamr** [reply]
Hi @ShaMan123,

Thank you very much for the reply. 
And, It's working when importing eraser_brush.mixin to the index.js as you said. Now I am able to build a fabric.js file with EraserBrush. 

But, I am receiving another error when I am going to use it. Do you have an idea about it?

Here is the issue: [https://github.com/fabricjs/fabric.js/discussions/8339](https://github.com/fabricjs/fabric.js/discussions/8339)

Thanks.
Dilanka Rathnayake.

**Comment by ShaMan123** [comment]
As mentioned master is WIP.
And eraser is broken there.
So I suggest using the 5.x branch.
Eraser will probably be the last thing to migrate since it depends on prototype mutation that might be refactored

**Reply by dilankamr** [reply]
Thank you very much for the suggestion. I used the 5.24 branch with merging 5.x branch and the Eraser is working fine there, Thanks again... :)
